### ISSUE TITLE
A question about kvstore and optimizer.

### ISSUE THREAD

**Issue by wanderingpj** [dmlc/mxnet#4091]
Is optimizer.py only executed on server node? 
It seems the server node gives every node the same learning rate.  How do other nodes get the learning rate from the server node ? From kvstore ? 
If we want different node training with different learning rate , how to do it?

**Comment by feiyulv** [comment]
Worker node only compute grads, and send them to server. The server node collects grads from workers and merge them to model, then pull the new model from server. Learning rate is not needed for workers.

**Comment by wanderingpj** [comment]
So we cant't make nodes train with different speed? @feiyulv

**Comment by mli** [comment]
there is a `lr_scheduler`, which decides how to change the learning rate. you can has a customized scheduler which gives different lr for different nodes

**Comment by wanderingpj** [comment]
thank you@mli
